src/applications/vaos/appointment-list/pages/CancelAppointmentPage/CancelPageLayout.jsx

```diff
@@ -113,11 +113,11 @@ export default function CancelPageLayout() {
               </>
             )}
           {!!facility && (
-            <>
+            <span data-dd-privacy="mask">
```

Comment:
If we are masking facility info on the cancel page, we should also mask the clinic info below. Same with practitionerName (which corresponds to provider name) above?

Reply:
Also there's another section with facility information at ~line 164.

Reply:
Hmmm I see, but then why are we masking this facility information section here? I suppose we should mask facility/clinic information consistently across all details, cancel and confirmation pages.